src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/PackageManagerTopPanel.xaml.cs

```diff
@@ -13,14 +18,145 @@ namespace NuGet.PackageManagement.UI
     /// </summary>
     public partial class PackageManagerTopPanel : UserControl
     {
-        private FilterLabel _selectedFilter;
+        private TabItem _selectedTabItem
+        {
+            get
+            {
+                return tabsPackageManagement.SelectedItem as TabItem;
+            }
+            set
+            {
+                tabsPackageManagement.SelectedItem = value;
+            }
+        }
+        public TabItem _tabConsolidate { get; private set; }
+        public Border _countConsolidateContainer { get; private set; }
+        public TextBlock _countConsolidate { get; private set; }
```

Comment:
The naming convention used here is non-standard.  These members should be named `SelectedTabItem`, `TabConsolidate`, `CountConsolidateContainer`, and `CountConsolidate`, respectively.

Reply:
Here's [the XAML on Dev branch](https://github.com/NuGet/NuGet.Client/blob/dev/src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/FilterLabel.xaml#L54) where the same controls are prefixed with _. In a discussion, it was mentioned that WPF turns these into private fields, therefore fall under the convention of an _ prefix.

Or are you suggesting I rename the existing XAML controls as well?

Reply:
But they are no longer private fields with your change.  You are introducing public properties.  Rename the controls, if necessary.